forward/forward.go

```diff
@@ -170,7 +170,7 @@ func (f *Forwarder) cloneRequest(req *http.Request, u *url.URL) (*http.Request,
 	// We need to make sure the host is defined in the URL (not the actual URI)
 	outReq.URL.Host = req.Host
 	// Make sure we define an opaque URL, so the URI is just the path
-	outReq.URL.Opaque = req.URL.Path
+	outReq.URL.Opaque = req.URL.EscapedPath()
```

Comment:
`req.URL.Path` is always in decoded format, by assigning it to `Opaque`, we altered the request path if it is in not all Latin characters, hence caused https://github.com/getlantern/http-proxy/issues/70.

Though this change fixes the issue user reported, it will still break if the requested URI is not encoded. Go client and Chrome will always encode the URL, while `curl` will not.


https://tools.ietf.org/html/rfc2616#section-5.1.2 says

> A transparent proxy MUST NOT rewrite the "abs_path" part of the
   received Request-URI when forwarding it to the next inbound server,
   except as noted above to replace a null abs_path with "/".

`http-proxy` is not transparent but it did cause issue.

 @uaalto What's the purpose of replacing `URL.Opaque`? We may find out a more robust solution to fix #70.